packages/common/src/ui/components/navigation/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,10 @@
+import MuiTabs from '@material-ui/core/Tabs';
+import { styled } from '@material-ui/system';
+
+export const Tabs = styled(MuiTabs)({
+  minHeight: '42px',
+  maxHeight: '42px',
```

Comment:
trying to reduce the gap between the indicator highlight and the text - couldn't find another way!